packages/shared/lib/services/sync/orchestrator.service.ts

```diff
@@ -61,14 +62,58 @@ interface CreateSyncArgs {
     syncName: string;
 }
 
+const logger = getLogger('orchestrator.service');
+
 export class OrchestratorService {
-    public async create(
+    public async createSyncForConnection(nangoConnectionId: number, logContextGetter: LogContextGetter, orchestrator: Orchestrator): Promise<void> {
```

Comment:
this is the `initiate` function moved from `syncClient` and renamed `createSyncForConnection`.
I wanted to merge it with the `create` function (renamed `createSyncForConnections`) but the input types are too different and I would have need to refactor the connectionCreatedHook too much. So until I find the courage to do it those 2 functions will remain. At least now they live next to each other